database.go

```diff
@@ -0,0 +1,1235 @@
+// This file is part of goBizFly
+
+package gobizfly
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+)
+
+const (
+	databaseInstancesResourcePath      = "/instances"
+	databaseNodesResourcePath          = "/nodes"
+	databaseTasksResourcePath          = "/tasks"
+	databaseEnginesResourcePath        = "/engines"
+	databaseConfigurationsResourcePath = "/configurations"
+	databaseBackupsResourcePath        = "/backups"
+	databaseSchedulesResourcePath      = "/schedules"
+	databaseAutoscalingResourcePath    = "/autoscaling"
+	databaseTrustedSourcesResourcePath = "/trusted_sources"
+)
+
+var _ DatabaseService = (*databaseService)(nil)
+
+type databaseService struct {
```

Comment:
Just have cloud database service, haven't database service. so we shouldn't write database service.